app/components/alert-prompt/component.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import Modal, { ModalProps } from 'components/modal';
+import Button from 'components/button';
+import Icon, { IconProps } from 'components/icon';
+
+export interface AlertPromptProps {
+  /**
+   * Title of the prompt
+   */
+  title: string;
+  /**
+   * Description displayed below the title
+   */
+  description?: string;
+  /**
+   * Whether the prompt is diplayed
+   */
+  open: ModalProps['open'],
+  /**
+   * Whether the user can close the prompt without having to accept nor refuse
+   */
+  dismissible?: ModalProps['dismissable'],
+  /**
```

Comment:
Great to see the properties documented 👏 
We should do the same in all components from now on 💯